js/src/views/Application/TabBar/tabBar.js

```diff
@@ -188,7 +188,7 @@ class TabBar extends Component {
     return (
       <ToolbarGroup>
         <div className={ styles.logo }>
-          <img src={ imagesEthcoreBlock } />
+          <img src={ imagesEthcoreBlock } height={ 28 } />
```

Comment:
Only because the CSS is loaded in JS, so there is a flash of ugly unstyled content on load. The logo was the thing that upset me the most.
With a bit more work, we should ship CSS files in production env anyway. This is just a quicker fix